MANIFEST.in

```diff
@@ -1,4 +1,4 @@
-include README.rst LICENSE NOTICES HISTORY.txt pipenv/patched/safety.zip
+include README.rst CONTRIBUTING.rst LICENSE NOTICES HISTORY.txt pipenv/patched/safety.zip
```

Comment:
We don't want to distribute non-essential documentation in our wheels. Let's remove this since new contributors can find it here now.